rpc/tests/rpc_command_tests/get_blocks_in_chain.rs

```diff
@@ -0,0 +1,57 @@
+use super::*;
+use commands::GetBlocksInChain;
+use responses::BlocksInChainResponse;
+use std::str::FromStr;
+
+#[tokio::test]
+async fn get_blocks_in_chain_ok() {
+    let chain_id = get_chain_id_string();
+    let command = generate_boxed_get_blocks_command(chain_id);
+
+    let client = get_rpc_client();
+    assert!(client.check_node_online().await);
+
+    let raw_response = client.execute(&command).await;
+    assert!(raw_response.is_ok());
+
+    let response = raw_response.unwrap();
+    assert_eq!(response.status(), 200);
+
+    let response_text = response.text().await.unwrap();
```

Comment:
is the await here necessary?

Reply:
yep; no such thing as an unnecessary await in rust since you can't await non-futures (nice!)